packages/backend-proxy-middleware/README.md

```diff
@@ -0,0 +1,149 @@
+#  `@sap-ux/backend-proxy-middleware`
+
+The `@sap-ux/backend-proxy-middleware` is a [Custom UI5 Server Middleware](https://sap.github.io/ui5-tooling/pages/extensibility/CustomServerMiddleware) for proxying requests to backend servers. The middleware is agnostic to running in local environments and SAP Business Application Studio. Additionally, if credentials are maintained with the SAP Fiori tools then these will be used.
+
+It can be used either with the `ui5 serve` or the `fiori run` commands.
+
+## Configuration Options
+
+### `backend`
+| Option        | Type | Description |
+| ------------- | ------------- | ----------- |
+| `url`         | `string` mandatory (local)  | Mandatory URL pointing to the backend system. *Not required if destination is provided and the proxy is running SAP Business Application Studio |
+| `destination` | `string` mandatory (if no url) | Required if the backend system is available as destination in SAP Business Application Studio. |
+| `destinationInstance` | `string` optional | If a destination needs to be read by a specific instance of a destination service then you need to provide the id of the service as optional property `destinationInstance`.|
+| `path`        | `string` mandatory     | Path that is to be proxied |
+| `pathPrefix`  | `string` optional      | If provided then the path will be replaced with the pathPrefix before forwarding |
+| `client`      | `string|number` optional      | sap-client parameter |
+| `scp`         | `boolean` optional      | If set to true the proxy will execute the required OAuth routine for the ABAP environment on SAP BTP |
+| `apiHub`      | `boolean` optional      | If set to true then the proxy will connect to the SAP API Business Hub
+| `proxy`       | `string` optional      | If set then it will override the proxy settings from node.
+
+Additional optional experimental property `bsp` (type `string`): The BSP property is only needed for the FLP Embedded Flow. The property refers to the BSP Application Name. In that case, we need to redirect the manifest.appdescr request to the local manifest.json in order to overwrite the deployed application with the local one.
+
+### `options`
+Optional object that can be used to directly set options of the used `http-proxy-middleware` modules (https://www.npmjs.com/package/http-proxy-middleware#http-proxy-middleware-options);
```

Comment:
Maybe we need to say that not all options are supported, because some of the options require defining a function. And we can't provide function as parameter in YAML.

Reply:
Yes, thanks!